Sources/TSCUtility/FSWatch.swift

```diff
@@ -47,7 +47,7 @@ public class FSWatch {
         self.paths = paths
         self.latency = latency
 
-      #if canImport(Glibc)
+      #if os(Linux) || os(Android)
```

Comment:
is this leaving anything on the floor? ie some other platform that can use glibc would now fatalError? I guess the if condition can be reordered to bring OpenBSD on top and then if else the glibc to bring it back?

Reply:
Platforms that may not support inotify canImport(Glibc) -- inotify appears to be a Linux kernel feature, not a libc feature. The only Linux kernel based systems that Swift supports that I'm aware of are Linux and Android, I don't think there are any others.

Reply:
Forgot to mention. I'll try bringing the platform specific condition up.

Reply:
Yes, this works. Thanks!